seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -37,12 +41,18 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
   }
 
   def send(msg: String): Unit = {
+    if (comClosed)
+      throw new ComJSEnv.ComClosedException
     awaitForBrowser()
-    browser.getWebDriver.executeAsyncScript(sendScript, msg)
+    val encodedMsg =
+      msg.replace("\u0007", "\u0007\u0007").replace('\u0000', '\u0007')
```

Comment:
I chose `"\u0007"` because it is a character that nobody would ever use in javascript. This was chosen to minimize the overhead.